### ISSUE TITLE
Why I don't have a logs file and loss print in the training process? 

### ISSUE THREAD

**Issue by wangtianlong1994** [google/automl#132]
`python main.py 
--training_file_pattern=tfrecord/train*
--model_name=efficientdet-d3 
--model_dir=tmp/model 
--hparams="use_bfloat16=false, num_classes=6"
 --use_tpu=False --ckpt=tmp/efficientdet-d3 
--validation_file_pattern=tfrecord/val* 
--val_json_file=tfrecord/annotations/val2017.json
--mode='train_and_eval'`

I want to use a tensorboard to check the training effect

> I0401 16:50:16.976712 140657036416832 tpu_estimator.py:2307] global_step/sec: 1.84509
> INFO:tensorflow:examples/sec: 7.38037
> I0401 16:50:16.976903 140657036416832 tpu_estimator.py:2308] examples/sec: 7.38037
> INFO:tensorflow:global_step/sec: 1.46901
> I0401 16:50:17.657439 140657036416832 tpu_estimator.py:2307] global_step/sec: 1.46901
> INFO:tensorflow:examples/sec: 5.87603
> I0401 16:50:17.657559 140657036416832 tpu_estimator.py:2308] examples/sec: 5.87603
> INFO:tensorflow:global_step/sec: 1.46089

**Comment by YinRui1991** [comment]
Do you solve it? How to see the loss during training process?

**Comment by wangtianlong1994** [comment]
> 你解决了吗？在培训过程中如何看待损失？

You can use tensorboard to see loss, cd to -- model_dir,

Enter the following command `tensorbboard -- logdir =. /`, which will open the` *. tfevents` file under the folder

**Comment by YinRui1991** [comment]
@wangtianlong1994  Thanks, it seems I have to modify the code to see the loss in the terminal(I can't use tensorboard limited by the port restriction)
